assessments/CLI_Circle/earthmover.yml

```diff
@@ -0,0 +1,589 @@
+        {'oa_category_7':{'math_v1_operations_v1': 'Math Operations'}}
+      ]
+    },
+    'math_opt_v1': {
+      'name': 'Optional Math',
+      'objective_assessments': [
+        {'oa_category_1':{'math_opt_v1_patterns_v1': 'Optional Math Patterns'}},
+        {'oa_category_2':{'math_opt_v1_real_world_v1': 'Optional Math Real World'}}
+      ]
+    },
+    'letter_sound_v1': {
+      'name': 'Letter-Sound Correspondence',
+      'objective_assessments': []
+    },
+    'story_retell_comp_v1': {
+      'name': 'Story Retell and Comprehension',
+      'objective_assessments': []
+    },
+    'book_print_v1': {
+      'name': 'Book and Print Knowledge',
+      'objective_assessments': []
+    },
+    'science_v1': {
+      'name': 'Science',
+      'objective_assessments': []
+    },
+    'social_studies_v1': {
+      'name': 'Social Studies',
+      'objective_assessments': []
+    },
+    'social_emotional_v1': {
+      'name': 'Social Emotional Behaviors',
+      'objective_assessments': [
+        {'oa_category_1':{'social_emotional_v1_positive_social_v1': 'Positive Social Behaviors'}},
+        {'oa_category_2':{'social_emotional_v1_classroom_community_safety_v1': 'Classroom Community and Safety'}},
+        {'oa_category_3':{'social_emotional_v1_emotion_behavior_v1': 'Emotion and Behavior Regulation'}},
+        {'oa_category_4':{'social_emotional_v1_self_care_v1': 'Self-Care'}},
+        {'oa_category_5':{'social_emotional_v1_approaches_to_learning_v1': 'Approaches to Learning'}}
+      ]
+    },
+    'early_writing_v1': {
+      'name': 'Early Writing Skills Early Writing Skills',
+      'objective_assessments': []
+    },
+    'approaches_to_learning_exp_v2': {
+      'name': 'Approaches to Learning Expanded',
+      'objective_assessments': [
+        {'oa_category_1':{'approaches_to_learning_exp_v2_initiative_curiosity_v1': 'Initiative and Curiosity'}},
+        {'oa_category_2':{'approaches_to_learning_exp_v2_flexibility_v1': 'Flexibility'}},
+        {'oa_category_3':{'approaches_to_learning_exp_v2_creativity_dramatic_play_v1': 'Art/Creativity and Dramatic Play'}}
+      ]
+    },
+    'physical_dev_health_v1': {
+      'name': 'Physical Development and Health',
+      'objective_assessments': [
+        {'oa_category_1':{'physical_dev_health_v1_fine_visual_motor_v1': 'Fine and Visual Motor'}},
+        {'oa_category_2':{'physical_dev_health_v1_gross_motor_v1': 'Gross Motor'}},
+        {'oa_category_3':{'physical_dev_health_v1_health_v1': 'Health Status'}}
+      ]
+    },
+    'language_communication_2016_v1': {
+      'name': 'Speech Production and Sentence Skills',
+      'objective_assessments': []
+    },
+    'motivation_to_read_2016_v1': {
+      'name': 'Motivation to Read - En',
+      'objective_assessments': []
+    },
+    'cpm_admin_mode': {
+      'name': 'Administration Mode',
+      'objective_assessments': []
+    }
+  }
+%}
+
+{% set subjects_spa = {
```

Comment:
This makes a lot of sense, I just made these dictionaries more uniform. I also added a language indicator as part of the dict that indicates whether the assessment was taken in Spanish or English:

```
    'rln_v1': {
      'colname': 'rln_v1',
      'descriptor': 'Rapid Letter Naming',
      'objective_assessments': [],
      'subject_assess_lang': 'English',
      'subject_has_cp_col': True
    },
```

One thing to note is that CLI CIRCLE already brings in an "Assessment_Language" column, but the values inside this column are "En and Sp" and "English". Not sure why the Spanish subjects are labeled as "En and Sp"...part of me thinks it makes more sense to not use this column (given its values) and instead use the col I created in the dictionary for `AdministrationLanguage`, but not sure if it's better to just use what originally came from the data instead?...

Reply:
my suggestion wasn't very clear, but what I was picturing was _not_ having two dictionaries, one for english, one for spanish -- but having one dictionary, with a key for `spanish_colname`. are there cases where this is difficult because e.g. English and Spanish have different obj assess?